Desktop_Interface/isobuffer.cpp

```diff
@@ -383,31 +393,44 @@ double isoBuffer::getDelayedTriggerPoint(double delay)
     auto isValid = [=](uint32_t index)->bool
     {
         if (m_back > delaySamples)
-            return (index < ((uint32_t)m_back - delaySamples)) || (index >= (uint32_t)m_back);
+            return (index < m_back - delaySamples) || (index >= m_back);
         else
-            // Fixme: There's probably an off by one here.
-            return (index < (m_bufferEnd + m_back - delaySamples)) && (index >= m_back);
+            return (index < m_bufferLen + m_back - delaySamples) && (index >= m_back);
     };
 
     auto getDelay = [=](uint32_t index)->double
     {
         if (m_back > index)
             return (m_back - index) / static_cast<double>(m_samplesPerSecond);
         else
-            return (m_bufferEnd + m_back - index) / static_cast<double>(m_samplesPerSecond);
+			// NOTE: Probably an off-by-one but I guess it's not too bad since it's being put into a floating point value?
```

Comment:
I just ran through this (after a full day of programming, so I would not take my word as Gospel) and it looks like it should be OK.